src/rust/plugin-registry/src/static_files/hax_docker_plugin.nomad

```diff
@@ -0,0 +1,109 @@
+variable "plugin_id" {
+  type        = string
+  description = "The ID for this plugin."
+}
+
+variable "tenant_id" {
+  type        = string
+  description = "The tenant's ID. Used in the plugin-execution-sidecar."
+}
+
+variable "plugin_artifact_url" {
+  type        = string
+  description = "The url that specifies which binary to run as the plugin."
+}
+
+variable "plugin_count" {
+  type        = number
+  default     = 1
+  description = "The number of instances of the plugin to run."
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "The account ID of the aws account that holds onto the plugin binaries."
+}
+
+variable "plugin_runtime_image" {
+  type        = string
+  description = "The container that will load and run the plugin"
+}
+
+job "grapl-plugin" {
+  datacenters = ["dc1"]
+  namespace   = "plugin-${var.plugin_id}"
+  type        = "service"
+
+  # We'll want to make sure we have the opposite constraint on other services
+  # This is set in the Nomad agent's `client` stanza:
+  # https://www.nomadproject.io/docs/configuration/client#meta
+  constraint {
+    attribute = "${meta.is_grapl_plugin_host}"
```

Comment:
Are you thinking that we'll have one cluster for plugins or that we'll have tenant-specific clusters?
Also I think `meta.is_grapl_plugin_host` would work here

Reply:
This idea is originally from the RFC which I blindly followed without thinking about it:
https://github.com/grapl-security/grapl-rfcs/blob/main/text/0000-plugins.md#nomad-job

But it's actually really interesting if we, say, had some Nomad agents that ran on a1 metal instances just for plugins, and other ones that ran on other types for regular services, for example

Reply:
The idea is to colocate tenants on the AWS hardware, which is why we rely so heavily on Firecracker.